featuregate/README.md

```diff
@@ -52,6 +52,21 @@ otelcol --config=config.yaml --feature-gates=gate1,-gate2,+gate3
 
 This will enable `gate1` and `gate3` and disable `gate2`.
 
+### Strict mode
+
+Features gate can be executed in strict mode as well. This mode uses the flag `--feature-gates-strict`.
+This flag is mutually exclusive flag with `--feature-gates`.
+
+The flag follows the same notation as the `--feature-gates` flag and is interpreted with additional rules by the collector:
```

Comment:
I don't think this corresponds to the implementation, right? The definition here https://github.com/open-telemetry/opentelemetry-collector/blob/53670a347f1c7c4e0399dc76095384743c6adaf2/otelcol/flags.go#L54-L56 is that of a boolean flag, so it does not follow the same notation.

Reply:
OK, the README needs an update indeed. On it.